app-misc/anki/files/23.12.1/ninja-gentoo-setup.patch

```diff
@@ -0,0 +1,80 @@
+This is an adapted patch from the games/anki FreeBSD port.
+
+It adapts the ninja(-compatible) build system to the Gentoo system and
+prevents repo sync via git.
+* Mimic ninja-utils.eclass behavior to respect NINJA_VERBOSE.
+  Can also be implemented in the ebuild via runner args when the rest
+  of the patch gets dropped.
+* Respect NINJA_STATUS if already set.
+* Respect app-alternatives/ninja symlink (n2 is not yet in the tree).
```

Comment:
FWIW, happy to consider a PR for [this](https://github.com/evmar/n2) separately.

Reply:
Once upstream tags a release, I'd take you up on this.